Assets/Scripts/Background/BeyondBorderResize.cs

```diff
@@ -0,0 +1,118 @@
+// Thanks to Loek van den Ouweland for scaling code explanation;
+// their article is where I came across a script with similar code for obtaining camera values and scaling a background.
+// van den Ouweland, L. (2019, October 9). STRETCH A UNITY SPRITE TO FILL THE SCREEN IN A 2D GAME (GAMEOBJECT, NOT UI-CANVAS). 
+// Loek van den Ouweland Software Engineering. Retrieved March 20, 2023, from 
+// https://www.loekvandenouweland.com/content/stretch-unity-sprite-to-fill-the-screen.html.
+// This code increases/decreases the size of the GameObject sprite so that it can cover the empty left/right background space that exists 
+// when the player is using a large/horizontal screen to play the game. The sizing is dependent on the distance between the assigned border's 
+// corresponding side (boolean right) and the same camera border side. NOTE: This script must be applied to 2 filler sprites (right and left) to 
+// fill all empty space to the left and right of the game screen. This script was created for tiled continuous sprites in mind, which is why 
+// the size of the sprite is altered, not the scale.
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BeyondBorderResize : MonoBehaviour
+{
+    private float cameraHeight;
+    private float cameraWidth;
+    private float spriteWidth;
+    private float spriteHeight;
+    private float sizeFactorWidth;
+    private float sizeFactorHeight;
+    private float borderWidth;
+    private float borderEdge;
+    private float cameraEdge;
+    private float localXScale;
+    private Vector2 currentSize;
+    private SpriteRenderer fillerSprite;
+
+    public bool right;  // true if filling the space for the right side of the screen, false if otherwise
+    public SpriteRenderer border;  // variable to store border sprite
+    
+    
+    void Awake()
+    {
+        cameraHeight = Camera.main.orthographicSize * 2;
+        cameraWidth = cameraHeight * Camera.main.aspect;
+        currentSize = new Vector2(cameraWidth, cameraHeight);
+        localXScale = transform.localScale.x;
+        borderWidth = border.size.x * border.transform.localScale.x;
+
+        if (!right) {
+            borderEdge = border.transform.position.x - (borderWidth / 2);
+            cameraEdge = Camera.main.transform.position.x - (cameraWidth / 2);
+        } else {
+            borderEdge = border.transform.position.x + (borderWidth / 2);
+            cameraEdge = Camera.main.transform.position.x + (cameraWidth / 2);
+        }
+        
+        fillerSprite = GetComponent<SpriteRenderer>();
+
+        // If background + edges combined are already wider than the camera, no need to resize sprite.

```

Comment:
I think this comment is a little inaccurate because it seems like the condition only checks if the border is outside of the camera, not the border and the resizable background. I'm not completely sure if that is what's meant by "background + edges" here, though.

Reply:
Good point, comment has been edited.